### PR TITLE
fix/remote-services-project-id

### PR DESCRIPTION
**Pull request by manubo**

### CODE REVIEW

**Comment by manubo** [comment]
[BROWSER-EXTENSION: Preselect Project in Clickup and Jira](https://trello.com/c/yRn6Mr3m/13633-browser-extension-preselect-project-in-clickup-and-jira)

File: src/js/remoteServices.js [hunk 1]
```diff
@@ -43,7 +43,29 @@ export default {
...
+          if (projectId) {
+            return projectId
+          }
+          const match = element.textContent.match(projectRegex)
+          if (match && match[1]) {
+            return match[1]
+          }
+          return null
+        }, null)
+    },
```
Comment (manubo) [inline src/js/remoteServices.js:68]: SCR-20231113-pvwf

File: src/js/remoteServices.js [hunk 2]
```diff
@@ -96,14 +118,27 @@ export default {
...
+    projectId: (document, service, { projectId }) => {
+      // The title of the issue
+      const projectIdFromIssue = projectIdentifierBySelector(
         "[data-testid='issue.views.issue-base.foundation.summary.heading']",
-      )(document) ||
-      projectIdentifierBySelector("[data-testid='software-board.header.title.container']")(
-        document,
-      ) ||
-      projectId,
+      )(document)
```
Comment (manubo) [inline src/js/remoteServices.js:125]: SCR-20231113-pwql

File: src/js/remoteServices.js [hunk 3]
```diff
@@ -96,14 +118,27 @@ export default {
...
+      }
+
+      // The second breadcrumb item
+      const match = document
+        .querySelector('nav[aria-label="Breadcrumbs"] ol li:nth-child(2)')
+        ?.textContent?.match(projectRegex)
+
+      if (match && match[1]) {
+        return match[1]
+      }
```
Comment (manubo) [inline src/js/remoteServices.js:138]: SCR-20231113-pwxs-3

File: src/js/remoteServices.js [hunk 4]
```diff
@@ -43,7 +43,29 @@ export default {
...
+    projectId: (document) => {
+      // The title of the task
+      const match = document
+        .querySelector(".cu-task-title__overlay")
+        ?.textContent?.match(projectRegex)
+      if (match && match[1]) {
+        return match[1]
+      }
+
+      // Breadcrumbs on top of the card, reverse order in order ot priritize the last item
```
Comment (sunsations) [inline src/js/remoteServices.js:55]: prioritize

**Pull request merged**